clab/config.go

```diff
@@ -557,3 +608,11 @@ func sysMemory(v string) uint64 {
 	}
 	return m
 }
+
+func parseLongName(input string) (string, string, string, error) {
+	result := strings.Split(input, "-")
+	if len(result) != 3 {
+		return "", "", "", fmt.Errorf("failed to parse container name %q", input)
+	}
+	return result[0], result[1], result[2], nil
+}
```

Comment:
I'm afraid this won't work
a lab/node name can have dashes, so you can have `clab-my-lab-my-node` full name
so the way to parse the long name you need to take in the lab name, and split on that